### ISSUE TITLE
Audio field does not display anything 

### ISSUE THREAD

**Issue by Giorgi625** [JoomCoder-com/JoomCCK#166]
Audio field works in submit form but it does not show anything in List or Full view. 
also there are problems with template settings if selected player in field settings. It popup is empty. if selected is list it has some options in popup but as a result there is still nothing shown in frontend.

**Comment by andiamine** [comment]
We will redo audio player using another player instead of JWPlayer.

**Comment by CMS-CORP** [comment]
> We will redo audio player using another player instead of JWPlayer.

will be good , now there are new players with secure feature

**Comment by ALLJSHARE** [comment]
> We will redo audio player using another player instead of JWPlayer.

yes need to improve too, we need it for ours modules too . some great Audio player OS  are available
@andiamine @LordHelmchen666
